Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -117,10 +117,6 @@ type Props = $ReadOnly<{|
  * source of truth.
  */
 class DatePickerIOS extends React.Component<Props> {
-  static DefaultProps: {|mode: $TEMPORARY$string<'datetime'>|} = {
-    mode: 'datetime',
-  };
```

Comment:
We still need to set the default value of this prop. So for every access of `props.mode`, if no value is set it still needs to be `datetime` -- the thing we want to remove is depending on `defaultProps` static type as a construct used in React.

You can take a look at destructuring with a default value here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
In addition, can you add a snapshot test that ensures the default value is used? You can take a look at an example component snapshot test from `VirtualizedList-test.js`: https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/Libraries/Lists/__tests__/VirtualizedList-test.js